commit bfaa6a99a2971cc0eda1b3588154ccef3edf7916
Author: Enrico Scholz <enrico.scholz@sigma-chemnitz.de>
Date:   Mon Sep 3 18:54:43 2012 +0200

    readahead: lowered required RAM size
    
    kernel reserves some space for itself so that readahead will not run
    on 128 MiB devices.  Lower the limit a little bit to make it available
    there.

diff --git a/src/readahead/readahead-common.c b/src/readahead/readahead-common.c
index 99dbac2..09764d2 100644
--- a/src/readahead/readahead-common.c
+++ b/src/readahead/readahead-common.c
@@ -155,7 +155,7 @@ bool enough_ram(void) {
         assert_se(sysinfo(&si) >= 0);
 
         /* Enable readahead only with at least 128MB memory */
-        return si.totalram > 127 * 1024*1024 / si.mem_unit;
+        return si.totalram > 120 * 1024*1024 / si.mem_unit;
 }
 
 int open_inotify(void) {
